src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -1384,6 +1384,13 @@
     <Resource Include="UI\Images\profile_normal.png" />
   </ItemGroup>
   <ItemGroup>
+    <Resource Include="UI\Images\watch_selected_48_48.png" />
+    <Resource Include="UI\Images\pin_48_48.png" />
+    <Resource Include="UI\Images\pin_default_48_48.png" />
+    <Resource Include="UI\Images\pin_hidden_48_48.png" />
+    <Resource Include="UI\Images\pin_selected_48_48.png" />
+    <Resource Include="UI\Images\watch_default_48_48.png" />
+    <Resource Include="UI\Images\watch_hidden_48_48.png" />
```

Comment:
These dont seem from your changes right?

Reply:
Yeah, some connector stuff might be included. I thought perhaps this had been brought into master already!